crates/codegen/schema/src/types/production.rs

```diff
@@ -83,6 +83,12 @@ impl<T> VersionMap<T> {
                 .and_then(|v| versions.get(v).unwrap().clone()),
         }
     }
+    pub fn get_versions(&self) -> Option<std::collections::BTreeSet<Version>> {
```

Comment:
looks like this is unused? I wonder why it didn't produce a warning though.

Reply:
didn't look at this crate. deleted method.